sigar_interface.go

```diff
@@ -353,19 +353,26 @@ type ProcIo struct {
 }
 
 type ProcMem struct {
-	Size        uint64
-	Resident    uint64
-	Share       uint64
-	MinorFaults uint64
-	MajorFaults uint64
-	PageFaults  uint64
+	Size          uint64
+	Resident      uint64
+	Share         uint64
+	MinorFaults   uint64
+	MajorFaults   uint64
+	PageFaults    uint64
+	PageFileBytes uint64
```

Comment:
Maybe comment that this is only seen in Windows?  (Or maybe my understanding is wrong)